clusterloader2/pkg/test/simpletestexecutor.go

```diff
@@ -71,10 +71,23 @@ func (ste *simpleTestExecutor) ExecuteStep(ctx Context, step *api.Step) []error
 			// index is created to make i value unchangeable during thread execution.
 			index := i
 			wg.Start(func() {
-				if err := ctx.GetMeasurementManager().Execute(step.Measurements[index].Method, step.Measurements[index].Identifier, step.Measurements[index].Params); err != nil {
+				summary, err := ctx.GetMeasurementManager().Execute(step.Measurements[index].Method, step.Measurements[index].Identifier, step.Measurements[index].Params)
+				if err != nil {
```

Comment:
Shouldn't we be printing summary even if measurement failed with error?

Reply:
Assuming that now function returns multiple summaries, summaries will be printed even if error occurs.